### PR TITLE
Tweak the cue template format

### PR DESCRIPTION
**Pull request by ryanzhang-oss**
Tweak the cue template format

### CODE REVIEW

File: pkg/plugins/cluster.go [hunk 1]
```diff
@@ -43,8 +30,10 @@ func GetWorkloadsFromCluster(ctx context.Context, namespace string, c client.Cli
 	for _, wd := range workloadDefs.Items {
 		tmp, err := HandleDefinition(wd.Name, syncDir, wd.Spec.Reference.Name, wd.Spec.Extension, types.TypeWorkload, nil)
 		if err != nil {
-			fmt.Printf("[WARN]handle template %s: %v\n", wd.Name, err)
+			fmt.Printf("[ERROR] hanlde workload template `%s` failed with error: %v\n", wd.Name, err)
```
Comment (wonderflow) [inline pkg/plugins/cluster.go:33]: warn is enough I think, template broken won't block other workload to run.

File: pkg/plugins/cluster.go [hunk 2]
```diff
@@ -68,8 +57,10 @@ func GetTraitsFromCluster(ctx context.Context, namespace string, c client.Client
 	for _, td := range traitDefs.Items {
 		tmp, err := HandleDefinition(td.Name, syncDir, td.Spec.Reference.Name, td.Spec.Extension, types.TypeTrait, td.Spec.AppliesToWorkloads)
 		if err != nil {
-			fmt.Printf("[WARN]handle template %s: %v\n", td.Name, err)
+			fmt.Printf("[ERROR] hanlde trait template `%s` failed with error: %v\n", td.Name, err)
```
Comment (wonderflow) [inline pkg/plugins/cluster.go:60]: WARN is enough

File: pkg/plugins/cluster_test.go [hunk 3]
```diff
@@ -32,69 +30,70 @@ var _ = Describe("DefinitionFiles", func() {
...
-				Usage:    "specify app image",
-			},
-			{
-				Name:    "port",
-				Type:    cue.IntKind,
-				Short:   "p",
-				Default: int64(8080),
-				Usage:   "specify port for container",
+	/*
+		deployment := types.Capability{
```
Comment (wonderflow) [inline pkg/plugins/cluster_test.go:34]: this test is useful, please keep it in this PR.

File: vela-templates/metrics.cue [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+#metrics: {
+	// +usage=format of the metrics, default as prometheus
+	// +short=f
+	format:  *"prometheus" | string
+	path:    *"/metrics" | string
+	scheme:  *"http" | string
+	enabled: *true | bool
+	port:    *8080 | >=1024 & <=65535 & int
+	// +usage= the label selector for the pods, default is the workload labels
+	selector?: [string]: string
```
Comment (wonderflow) [inline vela-templates/metrics.cue:17]: cool, I wonder how it works in command line?

**Review by wonderflow — APPROVED** [review]
Generally LGTM, please fix the CI.

File: pkg/plugins/cluster_test.go (hunk 3 above)
Comment (ryanzhang-oss) [reply pkg/plugins/cluster_test.go:34]: this doesn't work as it's testing against a github remote cue file that is not updated

**Pull request merged**